metadata-ingestion/tests/unit/test_schema_util.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+from datahub.ingestion.extractor.schema_util import avro_schema_to_mce_fields
+
+
+class SchemaUtilTest(unittest.TestCase):
```

Comment:
Thanks for adding! We should include a test for the 

type: [ "null", "string" ] case as well!

Reply:
Test added.